packages/react/src/table/table-pagination.tsx

```diff
@@ -64,6 +65,7 @@ const TablePagination: React.FC<TablePaginationProps> = ({
       total={totalPagination}
       animated={animated}
       onChange={handlePageChanged}
+      page={currentPage}
```

Comment:
How do you use this?

Reply:
This is used to avoid losing control of `page` when switching between tables. The pagination would look like this without `page={currentPage}`:
![test](https://us1.myximage.com/2022/06/23/8e7585ba80da77f1dc4d326d466c1063.gif)
Note that the pointer moves to the left arrow at first.